php-base/build-scripts/entrypoint.sh

```diff
@@ -22,6 +22,9 @@ set -xe
 
 /bin/bash /build-scripts/move-config-files.sh
 
+# Checks the nginx config. We need to do this after moving the files.
+/usr/sbin/nginx -t -c /etc/nginx/nginx.conf
```

Comment:
Do we actually need to move the config files again in this script (and thus check the nginx.conf again)?

It might have been left over from the xrt deployments when should no longer be used (unless we're still supporting them).

Reply:
Users are still using the `gcr.io/google-appengine/php` image, because it is the only way to customize the php runtime right now.

You can not do the same check in your customized `Dockerfile` because the config files are not there yet.